pkg/processor/build/runtime/python/test/python_test.go

```diff
@@ -103,7 +97,13 @@ func TestIntegrationSuite(t *testing.T) {
 		return
 	}
 
-	suite.Run(t, newTestSuite("python"))
-	suite.Run(t, newTestSuite("python:2.7"))
-	suite.Run(t, newTestSuite("python:3.6"))
+	for _, runtime := range []string{
+		"python",
+		"python:2.7",
+		"python:3.6",
+	} {
+		TestSuite := new(TestSuite)
+		TestSuite.runtime = runtime
+		suite.Run(t, TestSuite)
+	}
```

Comment:
?
I would fix this back to what it was before (re-add constructor with runtime argument and make private)
Why the change?

Reply:
When running tests from IDE, the right click on a specific test and run won't work and will exit with
```
testing: warning: no tests to run
PASS

Process finished with exit code 0

```

thought the previous way was neat, it simply didnt do the job of making it easy to run a single test.